tester/.gitignore

```diff
@@ -0,0 +1,5 @@
+**/.vscode/

```

Comment:
Why did you create another .gitignore? Keep it centralised, you only need one

Reply:
Well, can you delete it and move this to the other .gitignore? :p

Reply:
Yup. Forgot to do it ^^